.idea/misc.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Ok I will remove all files of the IDE. Should I do merge PR or just to open it and wait you to see it?